packages/services/api/src/modules/organization/resolvers.ts

```diff
@@ -251,6 +253,22 @@ export const resolvers: OrganizationModule.Resolvers = {
         organization: organizationId,
       });
 
+      const currentUser = await injector.get(AuthManager).getCurrentUser();
+      const jsonUpdatedFields = JSON.stringify({
+        name: input.name
+      });
+      await injector.get(AuditLogManager).createLogAuditEvent({
+        eventType: 'ORGANIZATION_SETTINGS_UPDATED',
+        eventTime: new Date().toISOString(),
```

Comment:
@kamilkisiela do we need to think about TZ here?

Reply:
@kamilkisiela  what do you think?

Reply:
not sure if all our dates today are UTC-0 if not then we should have TZ